src/containers/quiz/selectable-question-quiz-view/SelectableQuestionQuizView.tsx

```diff
@@ -0,0 +1,135 @@
+import { ChangeEvent, FC, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import ArrowBackIcon from '@mui/icons-material/ArrowBack'
+import ArrowForward from '@mui/icons-material/ArrowForward'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import { SxProps } from '@mui/material/styles'
+
+import AppButton from '~/components/app-button/AppButton'
+import QuizQuestion from '~/containers/quiz/quiz-question/Question'
+
+import { styles } from '~/containers/quiz/selectable-question-quiz-view/SelectableQuestion.styles'
+import { getQuestionStatus } from '~/containers/quiz/quiz-question/Question.constants'
+import { spliceSx } from '~/utils/helper-functions'
+
+import {
+  Question,
+  ButtonVariantEnum,
+  SizeEnum,
+  UseFormEventHandler
+} from '~/types'
+
+interface SelectableQuestionQuizViewProps {
+  questions: Question[]
+  handleInputChange: UseFormEventHandler<
+    Record<string, string | string[]>,
+    ChangeEvent<HTMLInputElement>
+  >
+  handleNonInputValueChange: (key: string) => (value: string | string[]) => void
+  answers: Record<string, string | string[]>
+  isEditable?: boolean
+  showCorrectAnswers?: boolean
+  showPoints?: boolean
+  showAnswersCorrectness?: boolean
+  sx?: { root?: SxProps; question?: SxProps }
+}
+
+const SelectableQuestionQuizView: FC<SelectableQuestionQuizViewProps> = ({
+  questions,
+  handleInputChange,
+  handleNonInputValueChange,
+  answers,
+  showCorrectAnswers = false,
+  showPoints = false,
```

Comment:
Same here

Reply:
Thanks for suggestion! I think showCorrectAnswers and showPoints are good for these cases.